hack/hook-me.sh

```diff
@@ -256,122 +256,144 @@ keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+    fi
 }
 
 usage(){
-  echo "==================================================================DISCLAIMER============================================================================"
-  echo "This scripts needs to be run against the KUBECONFIG of a seed cluster, please set your KUBECONFIG accordingly"
-  echo "You also need to set the \`ignoreResources\` variable in your extension chart to \`true\`, generate and apply the corresponding controller-installation"
-  echo "========================================================================================================================================================"
+    echo "==================================================================DISCLAIMER============================================================================"
+    echo "This scripts needs to be run against the KUBECONFIG of a seed cluster, please set your KUBECONFIG accordingly"
+    echo "You also need to set the \`ignoreResources\` variable in your extension chart to \`true\`, generate and apply the corresponding controller-installation"
+    echo "========================================================================================================================================================"
 
-  echo ""
+    echo ""
 
-  echo "===================================PRE-REQs========================================="
-  echo "\`host\` commands for DNS"
-  echo "\`docker\` https://www.docker.com"
-  echo "===================================================================================="
+    echo "===================================PRE-REQs========================================="
+    echo "\`host\` commands for DNS"
+    echo "\`docker\` https://www.docker.com"
+    echo "\`yq\` https://github.com/mikefarah/yq"
+    echo "===================================================================================="
 
-  echo ""
+    echo ""
 
-  echo "========================================================USAGE======================================================================"
-  echo "> ./hack/hook-me.sh <service e.g., gardener-extension-provider-aws> <extension namespace e.g. extension-provider-aws-fpr6w> <webhookserver port e.g., 8443> [<quic-server port, e.g. 9443>]"
-  echo "> \`make EXTENSION_NAMESPACE=<extension namespace e.g. extension-provider-aws-fpr6w> WEBHOOK_CONFIG_MODE=service start\`"
-  echo "=================================================================================================================================="
+    echo "========================================================USAGE======================================================================"
+    echo "> ./hack/hook-me.sh <service e.g., gardener-extension-provider-aws> <extension namespace e.g. extension-provider-aws-fpr6w> <webhookserver port e.g., 8443> [<quic-server port, e.g. 9443>]"
+    echo "> \`make EXTENSION_NAMESPACE=<extension namespace e.g. extension-provider-aws-fpr6w> WEBHOOK_CONFIG_MODE=service start\`"
```

Comment:
Some extensions require that the `GARDEN_KUBECONFIG` environment variable is also set as they need connection to the garden cluster. Should we also add this to this doc. 
It could also be added here: https://github.com/gardener/gardener/blob/c87bdd3aa145ce8c139712a4c595218922017f4a/hack/hook-me.sh#L427